### ISSUE TITLE
Generated `client/index.d.ts` file has type errors

### ISSUE THREAD

**Issue by suerta-git** [arthurfiorette/prisma-json-types-generator#75]
> Tested on prisma `4.10.1` and `4.11.0` with prisma-json-types-generator `2.4.0`.

There are type errors like:

```
Namespace '".../prisma/client/index".Prisma' has no exported member 'StringNullableFilter'

'".../prisma/client/index".Prisma' has no exported member named 'StringNullableWithAggregatesFilter'. Did you mean 'StringWithAggregatesFilter'?
```
and so on.

Can you have a look at this?

**Comment by arthurfiorette** [comment]
@suerta-git please, provide more information on how i can reproduce this behavior.

**Comment by suerta-git** [comment]
@arthurfiorette Sure, here is a demo you can check out: https://github.com/suerta-git/reproduce-json-types-error

**Comment by arthurfiorette** [comment]
Hey @suerta-git, thanks for your time doing the reproduction steps. Its now fixed at [v2.5.0](https://github.com/arthurfiorette/prisma-json-types-generator/releases/tag/v2.5.0)!
_Feel free to reopen this PR in case of any future bugs :)_
